corehq/apps/accounting/tasks.py

```diff
@@ -771,7 +773,7 @@ def get_accounts_with_customer_invoices_over_threshold(today):
             invoices = [invoice for invoice in invoices if invoice.subscriptions.first().plan_version == plan]
             total_overdue_to_date = sum(invoice.balance for invoice in invoices)
 
-            if total_overdue_to_date > 100:
+            if total_overdue_to_date >= UNPAID_INVOICE_THRESHOLD:
```

Comment:
@gbova I changed ```>``` to ```>=``` for consistency.  Let me know if using ```>``` was intentional.

Reply:
`>=` seems fine to me